src/main/java/com/app/models/menuactions/DeleteItemMenuAction.java

```diff
@@ -10,6 +12,6 @@ public DeleteItemMenuAction(String description) {
 
   @Override
   public void run() {
-    new ItemController().deleteItem();
+    new ItemController().deleteItem(new Scanner(System.in));
```

Comment:
Store the scanner in a variable to improve the readability.

Reply:
have added this change